src/Shared/Recruit.Vacancies.Client/Domain/Repositories/IBlockedOrganisationQuery.cs

```diff
@@ -0,0 +1,11 @@
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using Esfa.Recruit.Vacancies.Client.Domain.Entities;
+
+namespace Esfa.Recruit.Vacancies.Client.Domain.Repositories
+{
+    public interface IBlockedOrganisationQuery
+    {
+        Task<List<BlockedOrganisation>> GetByOrganisationIdAsync(string organisationId);
```

Comment:
Wouldn't we also want to pass in an organisation 'type' along with this? E.g, `Employer` or `Provider`?